### ISSUE TITLE
[Adaptive Card > Application Login]: Voice Over focus is not landing on the first element of the opened screen when activating the 'ApplicationLogin' option.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#7596]
### Target Platforms

iOS

### SDK Version

Version 1.0 (2.4.0-beta.20220628.3)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS Version: 15.5
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Navigate to "V1.5" control and select it.
4. Navigate on "V1.5" screen and select "Scenarios" button.
5. 'Scenario' screen will appear.
6. Navigate to "ApplicationLogin.json" control and double tap to activate it and observe the issue.

**Actual Result:**
While navigating to the 'ApplicationLogin.Json' option, on activating it screen reader focus is not landing on the activated control. It is just remain available on the applicationlogin.json option.

**Expected Result:**
After activating the 'ApplicationLogin.Json' option on activating it voice over focus should be landing on the activated control i.e. **'Name button'**.

**User Experience:**
It will impact the screen reader user if the screen reader focus does not lands on the activated control as it will take extra effort for the user.

**MAS Reference:**
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/03.02.01

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/176609153-fbf85ca6-54e6-46e5-bb9f-af816369f36e.MP4

### Card JSON

```JSON
NA
```
